disnake/http.py

```diff
@@ -287,9 +303,7 @@ async def request(
                 self._locks[bucket] = lock
 
         # header creation
-        headers: Dict[str, str] = {
-            "User-Agent": self.user_agent,
-        }
+        headers: Dict[str, str] = {}
```

Comment:
Is there a reason to remove this?

Reply:
it's part of the client session now, so we'd be overwriting it every request.

Reply:
[`4781471` (#959)](https://github.com/DisnakeDev/disnake/pull/959/commits/47814710cc6b74371b48106389ae651e548350ef)